docs/about/whats-new.ipynb

```diff
@@ -472,6 +560,21 @@
     "</div>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "**New in 0.8**\n",
+    "    \n",
+    "More creation functions were added:\n",
+    "\n",
+    "- Added `full` and `full_like`.\n",
+    "\n",
+    "</div>"
+   ]
+  },
```

Comment:
Maybe merge this with the 'New in 0.7' box and add the version at the end or beginning of each item. It would get pretty long with lots of 'biolerplate' if we keep adding stuff to lists in this way.

Reply:
I think we will just drop this once it gets too long.